src/main/java/org/blockchain_innovation/factom/identiy/did/LowLevelIdentityClient.java

```diff
@@ -224,4 +215,21 @@ public CommitAndRevealEntryResponse update(ReplaceKeyIdentityChainEntry updateEn
     public CommitAndRevealEntryResponse deactivate(DeactivateFactomDIDEntry deactivateEntry, Address ecAddress) {
         return getEntryApi().commitAndRevealEntry(deactivateEntry.toEntry(Optional.empty()), ecAddress).join();
     }
+
+    private String getChainIdFrom(String identifier) throws ParserException {
+        String chainId = identifier;
+        if (identifier.startsWith("did:factom:")) {
```

Comment:
Potential Nullpointer dereference (it is directly called by the public method which also has the identifier as argument

Reply:
Updated.